README.md

```diff
@@ -146,6 +146,8 @@ The source file is specified with `src` and the destination directory with `dst`
 
 * `parallelism`: *Optional. Default `10`* This int limit the number of concurrent operations Terraform will perform. See the [Terraform docs](https://www.terraform.io/docs/cli/commands/apply.html#parallelism-n) for more information.
 
+* `lock_timeout`: *Optional. Default `0`* Duration is seconds to retry a state lock. See the [Terraform docs](https://www.terraform.io/cli/commands/apply#lock-timeout-duration) for more information.
```

Comment:
The format accepted by the Terraform `--lock_timeout` flag includes a time unit like `3s`. But this field automatically adds the `s` even though the name is still called `lock_timeout`. Either remove the `s` or change the name to `lock_timeout_seconds` to avoid confusion.